cocos/2d/CCNode.h

```diff
@@ -38,6 +38,10 @@
 #include "2d/CCComponentContainer.h"
 #include "2d/CCComponent.h"
 
+#ifdef CC_STUDIO_ENABLED_VIEW   // for cocostudio only
+#include "CocosStudioExtension.h"
+#endif
+
```

Comment:
Cocos Studio define a data structure of resource description named "ResouceData" in this header file.